presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/RuleAssert.java

```diff
@@ -143,6 +143,10 @@ public void matches(PlanMatchPattern pattern)
     {
         return transaction(transactionManager, accessControl)
                 .singleStatement()
-                .execute(session, transactionSessionConsumer);
+                .execute(session, session -> {
+                            session.getCatalog().ifPresent(catalog -> metadata.getCatalogHandle(session, catalog));
```

Comment:
How does this set anything? Does metadata.getCatalogHandle have side-effects (ugh :/)?